indico/modules/events/abstracts/blueprint.py

```diff
@@ -42,6 +44,8 @@
 _bp.add_url_rule('/abstracts/<int:abstract_id>/abstract.pdf', 'display_abstract_pdf_export', RHDisplayAbstractExportPDF)
 _bp.add_url_rule('/abstracts/<int:abstract_id>/attachments/<file_id>/<filename>', 'download_attachment',
                  RHAbstractsDownloadAttachment)
+_bp.add_url_rule('/abstracts/mine', 'my_abstracts', RHMyAbstracts)
+_bp.add_url_rule('/abstracts/my-abstracts.pdf', 'my_abstracts_pdf_export', RHMyAbstractsExportPDF)
```

Comment:
Good point. While I'd usually comment if the filenames didn't match maybe this would be a case where it's not too bad to have `/mine.pdf` in the URL to be consistent with `/mine` but `my-abstracts.pdf` in the `send_file()` call so if someone saves it he still gets it as `my-abstracts.pdf`